src/annotator/guest.ts

```diff
@@ -435,7 +435,7 @@ export class Guest
     this._listeners.add(this.element, 'mouseup', event => {
       const { clientX, clientY, metaKey, ctrlKey } = event;
       const tags = annotationsAtPoint(clientX, clientY);
-      if (tags.length && this._highlightsVisible) {
+      if (tags.length) {
```

Comment:
The `_highlightsVisible` check is no longer needed because `annotationsAtPoint` now only returns visible highlights.